tests/conftest.py

```diff
@@ -0,0 +1,152 @@
+import datetime
+from fauxfactory import gen_string
+import pytest
+from testfm.advanced import Advanced
+from testfm.constants import (
+    DOGFOOD_ACTIVATIONKEY,
+    DOGFOOD_ORG,
+    katello_ca_consumer,
+    RHN_PASSWORD,
+    RHN_USERNAME,
+    RHN_POOLID,
+    upstream_url,
+)
+from testfm.log import logger
+
+
+@pytest.fixture(scope='function')
+def setup_for_hammer_defaults(request, ansible_module):
```

Comment:
@jameerpathan111 please add doc strings for each fixture. It will help others to understand which fixture is doing what.

Reply:
thanks, will add doc strings.